llvm/lib/Target/BPF/BPFISelLowering.cpp

```diff
@@ -65,10 +65,11 @@ BPFTargetLowering::BPFTargetLowering(const TargetMachine &TM,
 
   setOperationAction(ISD::BR_CC, MVT::i64, Custom);
   setOperationAction(ISD::BR_JT, MVT::Other, Expand);
-  setOperationAction(ISD::BRIND, MVT::Other, Expand);
```

Comment:
So, this does remove restriction to not produce indirect jumps?

Is there a way to control if we want to generate indirect jumps "in general" vs., say,  "only for large switches"? (Or even only for a particular switch?)

Reply:
I added an option to control how many cases in a switch statement to use jump table. The default is 4 cases. But you can change it with additional clang option, e.g., the minimum number of cases must be 6, then
```
clang ... -mllvm -bpf-min-jump-table-entries=6
```

I checked other targets, there are no control for a specific switch. So I think we do not need them for now.

Reply:
Awesome, thanks!